python_modules/dagster/dagster/_core/definitions/freshness_checks/time_partition.py

```diff
@@ -22,8 +22,8 @@ def build_time_partition_freshness_checks(
     deadline_cron: str,
     timezone: str = DEFAULT_FRESHNESS_TIMEZONE,
     severity: AssetCheckSeverity = DEFAULT_FRESHNESS_SEVERITY,
-) -> Sequence[AssetChecksDefinition]:
-    r"""For each provided time-window partitioned asset, constructs a freshness check definition.
```

Comment:
did all the partitions use to match? does this change anything?

Reply:
I added a test for this case and still works.